Function-Specification.md

```diff
@@ -84,30 +88,30 @@ Each block SHOULD include following information:
    4. Tutorial
    5. Exam (Dependent on UCL API Functionalities)
 
-> TODO: Does UCL Timetable include exam timetables?
+> TODO: Does UCL Timetable include exam timetables?   
```

Comment:
If I recall correctly UCL API does not include exam timetables. Exam timetable related content should be removed.